<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SightlineMappings</class>
 <widget class="QFrame" name="SightlineMappings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sightline mappings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0">
   <item>
    <widget class="QWidget" name="widgetPlot" native="true"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Sightline mappings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Geometry file:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="tbGeometryFile">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnBrowse">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lblCalcStatus">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="format">
         <string>%v/%m</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnCalc">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Calculate radial mapping</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnSave">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Save radial mapping</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnPlotXS">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Plot over cross-section</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>SightlineMappings</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>611</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
